tests/test_modules/test_model_contracts/test_hf_text_classification.py

```diff
@@ -251,3 +251,32 @@ def test_load_CLINC150_dataset(clinc_text_config):
     assert "NO_INTENT" in labels_str and "oos" not in labels_str
     # only 3 classes in the subset: "transfer", "transactions" and "NO_INTENT"
     assert len(ds) == 5
+
+
+def test_long_utterances_truncated(simple_text_config):
+    mod = HFTextClassificationModule(
+        DatasetSplitName.eval,
+        simple_text_config,
+        mod_options=ModuleOptions(
+            pipeline_index=0, model_contract_method_name=SupportedMethod.Predictions
+        ),
+    )
+    batch = Dataset.from_dict(
+        {
+            simple_text_config.columns.text_input: ["Hello " * 1000],
```

Comment:
If you are annoyed by the trailing space that this will create, here is a solution, but it's not important:
```suggestion
            simple_text_config.columns.text_input: " ".join(["Hello"] * 1000),
```

Reply:
Yeah it would not matter here. Thanks.